.eslintrc.json

```diff
@@ -7,14 +7,47 @@
     "prettier"
   ],
   "rules": {
-    "react/jsx-filename-extension": [1, { "extensions": [".js"] }],
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "object-curly-newline": ["error", {
         "ObjectExpression": "always",
         "ObjectPattern": { "multiline": true },
         "ExportDeclaration": { "multiline": true, "minProperties": 3 }
     }],
     "comma-dangle": ["error", {
-        "imports": "never"
-    }]
+      "arrays": "always-multiline",
```

Comment:
@ipheghe why are you writing new eslint rules in a task to implement login

Reply:
He already informed me about this, so no worries